examples/pytorch/eeg-gcnn/main.py

```diff
@@ -0,0 +1,218 @@
+import matplotlib.pyplot as plt
+import argparse
+import numpy as np
+import pandas as pd
+import torch
+import torch.nn as nn
+import torch.backends.cudnn as cudnn
+from sklearn.model_selection import train_test_split
+from joblib import load
+from EEGGraphConvNet import EEGGraphConvNet
+from EEGGraphDataset import EEGGraphDataset
+from dgl.dataloading import GraphDataLoader
+from torch.utils.data import WeightedRandomSampler
+from sklearn.metrics import roc_auc_score
+from sklearn.metrics import balanced_accuracy_score
+from collections import Counter
+
+
+if __name__ == "__main__":
+    # argparse commandline args
+    parser = argparse.ArgumentParser(description='Execute training pipeline on a given train/val subjects')
+    parser.add_argument('--num_feats', type=int, default=6, help='Number of features per node for the graph')
+    parser.add_argument('--num_nodes', type=int, default=8, help='Number of nodes in the graph')
+    parser.add_argument('--gpu_idx', type=int, default=0,
+                        help='index of GPU device that should be used for this run, defaults to 0.')
+    parser.add_argument('--num_epochs', type=int, default=10, help='Number of epochs used to train')
+    parser.add_argument('--exp_name', type=str, default='default', help='Name for the test.')
+    parser.add_argument('--batch_size', type=int, default=512, help='Batch Size. Default is 512.')
+    args = parser.parse_args()
+
+    # use GPU when available
+    _GPU_IDX = args.gpu_idx
+    _DEVICE = torch.device(f'cuda:{_GPU_IDX}' if torch.cuda.is_available() else 'cpu')
+    torch.cuda.set_device(_DEVICE)
+    print(f' Using device: {_DEVICE} {torch.cuda.get_device_name(_DEVICE)}')
+
+    # retrieve inputs
+    num_nodes = args.num_nodes
+    _NUM_EPOCHS = args.num_epochs
+    _EXPERIMENT_NAME = args.exp_name
+    _BATCH_SIZE = args.batch_size
+    num_feats = args.num_feats
+
+    # set up input and targets from files
+    memmap_x = f'norm_X'
```

Comment:
I checked the preprocessed dataset [here](https://figshare.com/articles/software/EEG-GCNN_Supporting_Resources_for_Reproducibility/13251452), which does not seem to have the file `'norm_X'`

Reply:
We haven't updated Figshare yet. We were dealing with a bug in the original code. So, some files need to be updated. We will do that next month, including norm_X. Also, norm_X is the normalized version of psd_featureds_data_X.

Reply:
In that case, we will need to wait for the files to be updated before we merge the PR.